src/Resources/views/Admin/Security/Resetting/reset.html.twig

```diff
@@ -48,6 +48,7 @@ file that was distributed with this source code.
                             'class': 'form-control',
                             'placeholder': 'form.new_password'|trans({}, 'FOSUserBundle')
                         }}) }}
+                        {{ form_errors(form.plainPassword.first) }}
```

Comment:
Please add this for the second as well

Reply:
I think there is not need, check RepeatedTypeValidatorExtension.

Reply:
It depends, what other validation was added by the developer.